src/core/import/csv/csv.spec.js

```diff
@@ -0,0 +1,34 @@
+import {Csv} from './csv';
+
+const csv = new Csv();
+
+describe('csv parser', () => {
+	it('should return empty array', () => {
```

Comment:
sorry, of course.
i meant that you should have more detailed sentence 
`'should return empty array wnhen input is not defined'`